### ISSUE TITLE
Salmon

### ISSUE THREAD

**Issue by willAubrecht** [ubccr/software-layer#280]
Hello,

Is it possible to install Salmon 1.4.0 (or latest) for RNA Sequencing research at Roswell?
Easybuild recipe located at https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/s/Salmon/Salmon-1.4.0-GCC-11.2.0.eb

Thank you for your help!

**Comment by tonykew** [comment]
I have a module built, which needs to go through internal testing.
I will update here when the module is available for production use.

**Comment by tonykew** [comment]
Salmon version 1.10.1 has been published to the ccrsoft/2023.01 software release

```
login2$ module spider salmon
----------------------------------------------------------------------------
  salmon: salmon/1.10.1
----------------------------------------------------------------------------
    Description:
      Salmon is a wicked-fast program to produce a highly-accurate,
      transcript-level quantification estimates from RNA-seq data.

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "salmon/1.10.1" module is available to load.

      gcc/11.2.0
[...]
login2$ module load gcc/11.2.0 salmon/1.10.1
login2$ salmon --help
salmon v1.10.1

Usage:  salmon -h|--help or 
        salmon -v|--version or 
        salmon -c|--cite or 
        salmon [--no-version-check] <COMMAND> [-h | options]

Commands:
     index      : create a salmon index
     quant      : quantify a sample
     alevin     : single cell analysis
     swim       : perform super-secret operation
     quantmerge : merge multiple quantifications into a single file
login2$ 
```

**Comment by willAubrecht** [comment]
Thank you all!
